mandatory/2-function-creation.js

```diff
@@ -6,7 +6,12 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num === "string") {return false}
```

Comment:
What would be the output for this line if I pass an object like
```
{foo: 1}
````

Reply:
false - because this is an object dictionary and not string

Reply:
and what would happen if I pass number 2
<img width="573" alt="Screenshot 2022-11-28 at 09 36 21" src="https://user-images.githubusercontent.com/40305104/204243826-625964c4-8267-446b-8dcf-c93748d8f477.png">

Reply:
my point was: would it make more sense to validate for a number than a string?

Reply:
is there any way I could pass to line 11 in the current code?

Reply:
awesome :) . First two line could even be joined together with an &&

Reply:
optimised to one line
return typeof num === "number" && num%2==0 && num <= 100 ||  num===100 ?  true : false

Reply:
do you ned num === 100 ? that is already included in num <= 100.
you could just retun this, without even the ternary statement.
`return typeof num === "number" && num%2==0 && num <= 100`

Reply:
thank you! Done!